jslvdan/coplant-app
# Modal Add Plant Component

## What I did

- Modal to Add plant
- Not working the image picker.
<img width="350" alt="Screen Shot 2020-09-02 at 14 45 26" src="https://user-images.githubusercontent.com/50433701/98610623-bf1eb480-22ce-11eb-8dc4-6fdf9af24753.png">

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feat/Modal-Add-Plant_component`
   1. `git pull origin feat/Modal-Add-Plant_component`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:19002

1. Check if the component looks just like the mockup in deferent screen sizes 

1. In VScode open `<relative path>`

1. Things to change in the code during the test
   ```jsx
   if (isAwesome){
     return <Component />
   }
   ```

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 4h30min

**Resolves:** #<issue number>
